tests/app/dao/test_notification_usage_dao.py

```diff
@@ -9,20 +10,64 @@
 
 def test_get_rates_for_year(notify_db, notify_db_session):
     set_up_rate(notify_db, datetime(2016, 4, 1), 1.50)
+    set_up_rate(notify_db, datetime(2016, 9, 1), 1.60)
     set_up_rate(notify_db, datetime(2017, 6, 1), 1.75)
-    rates = get_rates_for_year(datetime(2016, 4, 1), datetime(2017, 3, 31), 'sms')
-    assert len(rates) == 1
+    rates = get_rates_for_year(datetime(2016, 3, 31), datetime(2017, 3, 31), 'sms')
+    assert len(rates) == 2
     assert datetime.strftime(rates[0].valid_from, '%Y-%m-%d %H:%M:%S') == "2016-04-01 00:00:00"
     assert rates[0].rate == 1.50
-    rates = get_rates_for_year(datetime(2017, 4, 1), datetime(2018, 3, 31), 'sms')
-    assert len(rates) == 1
+    assert datetime.strftime(rates[1].valid_from, '%Y-%m-%d %H:%M:%S') == "2016-09-01 00:00:00"
+    assert rates[1].rate == 1.6
+    start_date, end_date = get_financial_year(2017)
```

Comment:
Could perhaps split this up into another test, or parametrize this to make it clearer as quite a lot going on here